docs/docsite/rst/collection_release.rst

```diff
@@ -0,0 +1,336 @@
+<https://ansible-collections.github.io/amazon.aws/branch/main/collections/amazon/aws/index.html>`_
+
+If you've not already installed these tools then you'll need to do so (this can be done in a virtual
+environment if desired):
+
+.. code-block:: bash
+   pip install ansible sphinx-ansible-theme antsibull-changelog antsibull-docs
+
+Add a release_summary changelog fragment
+----------------------------------------
+
+While not strictly necessary it's preferable to add a release summary that will be added to the
+changelog.  For example, the `release summary for 5.2.0
+<https://ansible-collections.github.io/amazon.aws/branch/stable-5/collections/amazon/aws/docsite/CHANGELOG.html#release-summary>`_
+
+**changelog/fragments/release-summary.yml:**
+
+.. code-block:: yaml
+  release_summary: |
+    Add a short description highlighting some of the key changes in the release
+
+Commit the release summary to your local branch.
+
+Generate the merged changelog
+-----------------------------
+
+Next we need to generate the merged changelog.  This will automatically delete the used fragments,
+update ``CHANGELOG.rst``, ``changelogs/changelog.yaml``, and ``git add`` what it changes.
+
+Commit all of these changes to your local branch.
+
+Create your Pull Request
+------------------------
+
+Once everything's been committed locally you can prepare a pull request.  The pull request should be
+for the relevant ``stable`` branch and **not** the ``main`` branch.
+
+All tests for the PR should pass prior to merging.  This pull request can be approved and merged as
+usual.
+
+Because ``CHANGELOG.rst`` is actually generated from ``changelogs/changelog.yaml``, if you need to
+fix issues with a changelog fragment, the easiest option is to revert the final changelog
+generation, fix the original fragment, and re-generate the final changelog (This is why you should
+commit small changes as you go).
+
+.. note::
+  Releases for amazon.aws should either be prepared by someone from the Anisble Cloud Content
+  team, or be approved by at least one person from the Ansible Cloud Content team.
+
+.. warning::
+  Prior to triggering the merge for the release pull request, please warn the `Ansible + AWS Matrix
+  chat room <https://matrix.to/#/#aws:ansible.com>`_ the final tagging (which releases the code to
+  `Ansible Galaxy <https://galaxy.ansible.com>`_) should be done using the pull request commit.
+
+Tag the release
+===============
+
+Pushing the release to `Ansible Galaxy <https://galaxy.ansible.com/>`_ is performed by Zuul.  When
+a tag is pushed to the repository Zuul will automatically build the relevant artifacts and push it
+to Galaxy.
+
+.. code-block:: bash
+
+  git fetch origin
+  git checkout stable-6
+  git reset --hard origin/stable-6
+  git tag 6.0.0
+  git push origin 6.0.0
+
+
+Announce the release
+====================
+
+Create a GitHub "Release"
+-------------------------
+
+Releases can be announced using the GitHub:
```

Comment:
Apologies, I'm not sure what you're saying here.  Given the links I think you might mean "the person doing the release should draft a release note on the repo"?